bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java

```diff
@@ -525,6 +525,18 @@ private void checkCameraConnection() {
             }
             return;// ffmpeg snapshot stream is still alive
         }
+
+        // if ONVIF cam also use connection state which is updated by regular messages to camera
+        if (thing.getThingTypeUID().getId().equals(ONVIF_THING) && snapshotUri.isEmpty()) {
+            logger.trace("is ONVIF with empty snapshot URI");
+            if (onvifCamera.isConnected()) {
+                logger.trace("cam is connected");
+                return;
+            } else {
+                logger.trace("cam not connected");
+            }
```

Comment:
We do not like logging PLUS changing the THING STATUS as this is doubling up the info. Please remove as the cameraCommunicationError() line changes the thing status and this is the preferred way to let users know.

Reply:
Ok, makes sense.